politeiawww/api/cms/v1/v1.go

```diff
@@ -531,7 +531,7 @@ type InvoicePayoutsReply struct {
 type PaymentInformation struct {
 	Token           string         `json:"token"`
 	Address         string         `json:"address"`
-	TxIDs           []string       `json:"txids"`
+	TxIDs           string         `json:"txids"`
```

Comment:
Comment should denote format of the string.  I'm guessing its a comma separated list of txids?  I don't think it'd be a bad idea to keep it as a slice of strings in order to make it easier to work with, but that's up to you.